### PR TITLE
Fix: #16380 support TestSuite redeploy option from UI

### LINKED ISSUE #16380: Redeploy all test suite IngestionPipelines from UI
image

post upgrade steps include redeploying Ingestion pipelines (in case we do any payload updates). However, these UI lists do not include test suites

We need to prepare a page for DQ pipelines and allow admins to redeploy them all

When we're done, update the docs accordingly

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

Fixes #16380 

Added support to re-deploy testSuite pipelines from UI:

https://github.com/open-metadata/OpenMetadata/assets/12962843/63ceb55f-c891-45ff-9609-6c19ef4074f8

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by ShaileshParmar11 — COMMENTED** [review]
Please add E2E test

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import test, { expect } from '@playwright/test';
+import { GlobalSettingOptions } from '../../constant/settings';
+import { createNewPage, redirectToHomePage } from '../../utils/common';
+import { settingClick } from '../../utils/sidebar';
+
+// use the admin user to login
+test.use({ storageState: 'playwright/.auth/admin.json' });
+
+test.describe('User with different Roles', () => {
+  test.beforeAll('Setup pre-requests', async ({ browser }) => {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts:22]: I believe we don't need this for spec as we are not doing any pre setup.

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    const { afterAction } = await createNewPage(browser);
+
+    await afterAction();
+  });
+
+  test.beforeEach('Visit user list page', async ({ page }) => {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts:28]:
```suggestion
  test.beforeEach('Visit home page', async ({ page }) => {
```

**Review by Sachin-chaurasiya — COMMENTED** [review]
Left a few comments, thanks

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts (hunk 2 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts]: Done thanks

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts (hunk 1 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TestSuitePipelineRedeploy.spec.ts:26]: Yes, updated

**Comment by chirag-madlani** [comment]
@Sachin-chaurasiya @ShaileshParmar11 please review

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**